queries.sql

```diff
@@ -8,3 +8,63 @@ SELECT (name, escape_attempts) FROM animals WHERE weight_kg > 10.50;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.40 AND 17.30;
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon%';
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+SELECT * FROM animals;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals
+WHERE birthdate > '2022-01-01';
+SAVEPOINT SP1;
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+SELECT * FROM animals;
+ROLLBACK TO SP1;
+SELECT * FROM animals;
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+SELECT COUNT(*) FROM animals;
+SELECT 
+   COUNT(escape_attempts) 
+FROM 
+   animals
+WHERE escape_attempts = '0';
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered, COUNT(*)
+FROM animals
+WHERE escape_attempts > 0
+GROUP BY neutered;
+SELECT
+       MIN(weight_kg),
+       MAX(weight_kg)
+FROM animals
+SELECT
+       AVG(escape_attempts)
+FROM animals
+WHERE date_of_birth >= '1990-01-01' AND date_of_birth <= '2000-12-31'
```

Comment:
- [ ] Here is how we can improve the query and output the right query results we are expection
```
   SELECT species, AVG(escape_attempts) FROM animals
   WHERE  date_of_birth >= '1990/01/01' AND date_of_birth <= '2000/12/31'
   GROUP BY species;

```
We are expected to answer the question below
![Screenshot from 2023-10-24 18-10-13](https://github.com/zhorabay/Vet-clinic/assets/46388113/09520683-8f7a-425d-95d3-775dd86dc7aa)

Reply:
Fixed.